src/App.tsx

```diff
@@ -1,24 +1,31 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
 import React from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { TodoProvider } from './components/TodoContext';
+import { USER_ID } from './utils/UserId';
+import { TodoHeader } from './components/TodoHeader';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoList } from './components/TodoList';
+import { ErrorNotification } from './components/ErrorNotification';
 
 export const App: React.FC = () => {
+  // const { todos } = useContext(TodoContext);
```

Comment:
Remove comment. Try it write obvious code that explains all we need without comments)